apps/web/pages/event-types/[type].tsx

```diff
@@ -660,7 +661,12 @@ const EventTypePage = (props: inferSSRProps<typeof getServerSideProps>) => {
                   <div className="flex">
                     <button
                       type="button"
-                      onClick={() => openLocationModal(location.type)}
+                      onClick={() => {
+                        locationFormMethods.setValue("locationType", location.type);
```

Comment:
Set the location so we actually edit the right place. As previously you were trying to edit a location type that isn't the one you clicked